mllib/src/main/scala/org/apache/spark/mllib/evaluation/RegressionMetrics.scala

```diff
@@ -31,13 +30,17 @@ import org.apache.spark.sql.DataFrame
  */
 @Since("1.2.0")
 class RegressionMetrics @Since("1.2.0") (
-    predictionAndObservations: RDD[(Double, Double)]) extends Logging {
+    predictionAndObservations: RDD[(Double, Double)], isUnbiased: Boolean = true)
```

Comment:
+1 for something along lines of regression through the origin.

I don't like `fitIntercept` since some regression models (like tree models) don't have a concept of intercept.